src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.scss

```diff
@@ -41,3 +41,10 @@ sam-datatable{
   overflow-y: scroll !important;
   overflow-x: scroll !important;
  }
+ .empty-results{
+  min-height: 350px;
+  max-height: 400px !important;
```

Comment:
@thomastighe Can you find a different way to get higher specificity without using `important`?